website/docs/pager.md

```diff
@@ -0,0 +1,225 @@
+---
+id: pager
+title: Pager
+slug: /apis/pager
+---
+
+`Pager`
+
+> TODO: add preview here
+
+## Usage
+
+```tsx
+import React from 'react';
+import { Dimensions, Image } from 'react-native';
+import {
+  GalleryItemType,
+  normalizeDimensions,
+  Pager,
+} from 'react-native-gallery-toolkit';
+
+const { height, width } = Dimensions.get('window');
+
+const pages: GalleryItemType[] = [
+  {
+    id: '1',
+    width: 400,
+    height: 300,
+    uri: 'https://placekitten.com/400/300',
+  },
+  {
+    id: '2',
+    width: 400,
+    height: 300,
+    uri: 'https://placekitten.com/400/300',
+  },
+  {
+    id: '3',
+    width: 400,
+    height: 300,
+    uri: 'https://placekitten.com/400/300',
+  },
+];
+
+export default function PagerBasicExampleScreen() {
+  return (
+    <Pager
+      pages={pages}
+      totalCount={pages.length}
+      keyExtractor={(item) => item.id}
+      initialIndex={0}
+      gutterWidth={20}
+      renderPage={(props) => {
+        const { targetWidth, targetHeight } = normalizeDimensions(
+          props.item,
+        );
+
+        return (
+          <Image
+            style={{ width: targetWidth, height: targetHeight }}
+            source={{ uri: props.item.uri }}
+          />
+        );
+      }}
+    />
+  );
+}
+```
+
+## Props
+
+### `initialIndex`
+
+Initial page index.
+
+type | default | required
+------ | ------ | ------
+`number` | `undefined` | YES
+
+### `initialDiffValue`
+
+@todo
```

Comment:
```suggestion
How many pages should be rendered on the first render. If the value 0 - it renders initially only a visible page. If the value is 1 and the index is 0 - renders only 2 pages, but if the index, for example, 1 - renders the first, second (active), and the third page.

Can be used to avoid some flickering on some high-end devices.
```